src/app/admin/services/controller/equipment-controller.service.ts

```diff
@@ -74,14 +75,30 @@ export class EquipmentController {
 
   private createRows(equipments: Equipment[]): TableRow[] {
     return equipments.map((equipment) => {
+      let actions = INITIAL_EQUIPMENT_ACTIONS_STATE;
       const categoryName = this.dictionaryService.getDictionaryValue(this.categoryDictionary, equipment.category);
       const statusName = this.dictionaryService.getDictionaryValue(this.statusDictionary, equipment.status);
       if (categoryName) equipment.categoryName = categoryName;
       if (statusName) equipment.statusName = statusName;
       if (equipment.status === this.statusIdsDictionary.archived) {
```

Comment:
let's move this logic to separae function. t.e. getActions ()getRowActions. then we can use only lst instad of let.